### ISSUE TITLE
Get ModelMetadata using gRPC

### ISSUE THREAD

**Issue by andreapairon** [kserve/modelmesh-serving#104]
I have the `example-tensorflow-mnist` predictor deploy and the `triton-2.x` pods running to serve the tensorflow model.

![image](https://user-images.githubusercontent.com/16152588/148932567-a8d672cd-9b95-4a2f-a789-a51ec74a7cda.png)

![image](https://user-images.githubusercontent.com/16152588/148932642-746e830c-5411-4ebb-a785-107fa2550561.png)

I'm trying to get the model metadata using gRPC but I'm facing issues:
- MODEL_NAME:  example-tensorflow-mnist

**Comment by pvaneck** [comment]
Thanks for the report, @andreapairon. I don't believe that the ModelMetadata endpoint is currently exposed by ModelMesh because of the need for the model to be loaded in order for the endpoint to work (in some cases, a model might be completely unloaded from the model servers due to lack of request activity). This might be something we can revisit and investigate how we can make this endpoint accessible. /cc @njhill

**Comment by andreapairon** [comment]
@pvaneck thank you for the reply. So, the only endpoint exposed is ModelInfer, right?

**Comment by njhill** [comment]
>  So, the only endpoint exposed is ModelInfer, right?

@andreapairon that's correct currently, but I think we could add support for ModelMetadata pretty easily, so will likely plan to do that soon.

For the `example-tensorflow-mnist` model you may also find the [quickstart example](https://github.com/kserve/modelmesh-serving/blob/main/docs/quickstart.md#3-perform-an-inference-request) helpful.
